go.mod

```diff
@@ -1,14 +1,15 @@
 module github.com/redhatinsights/yggdrasil-worker-package-manager
 
 go 1.21
+toolchain go1.22.5
```

Comment:
If we can update the base version of Go to 1.21, that should work. I added the explicit 1.20 version during the packit PR #22. https://github.com/RedHatInsights/yggdrasil-worker-package-manager/pull/22#pullrequestreview-2057988800

Reply:
See #31.

Reply:
The go still has problem with version 1.21, because the PR contains `toolchain go1.22.5`. Do we want to update go to version 1.22.5? I guess that we do not want it.

Reply:
zcalusic/sysinfo 1.1.0 requires Go 1.22, so unless we're willing to upgrade to 1.22, we have to keep the sysinfo pinned to an older version.